CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
 # Contributing to `code-quality-tools`
 
-First of all, thanks for your contribution to this project! ❤️ Here are some tips how to make your contributing efforts efficient and eventually accepted & merged.
+Before all anything, thanks for your contribution to this project! ❤️ Here are some tips how to make your contributing efforts efficient and eventually accepted & merged.
```

Comment:
Was that actually wrong…? 🤔 (Grammarly is OK with both.)

Reply:
This one is from stop-words, see https://github.com/sapegin/textlint-rule-stop-words and https://smartblogger.com/filler-words/, point 117. I also think that it is actually right. Maybe we should stop using this plugin.